frontend/src/app/features/work-packages/routing/wp-view-base/work-package-single-view.base.ts

```diff
@@ -101,7 +114,10 @@ export class WorkPackageSingleViewBase extends UntilDestroyedMixin {
       )
       .subscribe((wp:WorkPackageResource) => {
         this.workPackage = wp;
-        this.init();
+        if (!initialized) {
```

Comment:
Can't we check `this.workPackage` instead? Not sure how the initialized variable helps